views/pages/consent.njk

```diff
@@ -1,36 +1,171 @@
+        function signatureDraw() {
+            $('#signature_draw').show();
+            $('#signature_type').hide();
+        };
+
+        function signatureType() {
+            $('#signature_draw').hide();
+            $('#signature_type').show();
+        };
+
+        function switchSignatureMode() {
+            let value = $('input[type=radio][name=signature_mode]:checked')[0].value;
+            if(value === 'draw'){
+                signatureDraw();
+            }
+            else {
+                signatureType();
+            }    
+        };
+
+        function encodeSignatureDrawing() {
+            let data = $('#signature_pad').jSignature('getData', 'svg');
+            $('#signature_draw_data').val(data);
+        };
+
+        $(document).ready(function() {
+            let signaturePad = $("#signature_pad");
+            signaturePad.jSignature({width: 600, height: 200});
+
+            $('input[type=radio][name=signature_mode]').change(switchSignatureMode);
+            switchSignatureMode();
+
+            signaturePad.change(encodeSignatureDrawing);
+        });
+
+    </script>
+{% endblock %}
+
 {% block content %}
 
-    <h1>{{ __('section2.title') }}</h1>
+    <h1>{{ __('consent.title') }}</h1>
 
-    <p>{{ __('section2.preamble') | safe }}</p>
+    <p>{{ __('consent.preamble') }}</p>
+    
+    <p class="font-bold">{{ __('consent.may_contact_preamble') }}</p>
+    <ul>
+        <li>{{ __('consent.practioners') }}</li>
+        <li>{{ __('consent.facilities') }}</li>
+        <li>{{ __('consent.government') }}</li>
+        <li>{{ __('consent.compensation_boards') }}</li>
+        <li>{{ __('consent.employers') }}</li>
+        <li>{{ __('consent.insurers') }}</li>
+    </ul>
 
-    <form action="{{ route('consent.store') }}" method="post">
+    <form action="{{ route('consent.store') }}" method="post" class="w-full">
         <input type="hidden" name="_csrf" value="{{ _csrf }}">
 
-        {{ medicalConsent('consent', { '1':'section2.i_consent','0':'section2.i_dont_consent'}, data.consent, 'section2.consent_preamble', errors, {required: true}) }}
+        {{ 
+            checkBoxes(
+                'consent_optional_parties', 
+                {
+                    'education': 'consent.education', 
+                    'accountant':'consent.accountant',
+                    'financial': 'consent.financial',
+                    'volunteer': 'consent.volunteer',
+                    'employees': 'consent.employees'
+                }, 
+                data.consent_optional_parties if data.consent_optional_parties else [
+                    'education', 
+                    'accountant',
+                    'financial',
+                    'volunteer',
+                    'employees'
+                ], 
+                'consent.optional_parties_preamble', 
+                errors, 
+                {}
+            ) 
+        }}
+
+        <div class="border border-gray-600 border-solid bg-gray-100 p-8">
```

Comment:
Let's stick with `bg-gray-100`. It gives us AA with the red on gray, very close to AAA. Why are all the tailwind colours bluey-gray??